client/src/pages/automation/project/Project.tsx

```diff
@@ -381,33 +399,7 @@ const Project = () => {
         <>
             <LayoutContainer
                 className="bg-muted dark:bg-background"
-                leftSidebarBody={
-                    <>
-                        {!workflowIsRunning ? (
-                            workflowTestExecution?.job ? (
-                                <WorkflowExecutionDetailsAccordion
-                                    job={workflowTestExecution.job as JobModel}
-                                    triggerExecution={workflowTestExecution.triggerExecution as TriggerExecutionModel}
-                                />
-                            ) : (
-                                <div className="flex size-full items-center justify-center gap-x-1 p-3 text-muted-foreground">
-                                    <RefreshCwOffIcon className="size-5" />
-
-                                    <span>Workflow has not yet been executed.</span>
-                                </div>
-                            )
-                        ) : (
-                            <div className="flex size-full items-center justify-center gap-x-1 p-3">
-                                <span className="flex animate-spin text-gray-400">
-                                    <RefreshCwIcon className="size-5" />
-                                </span>
-
-                                <span className="text-muted-foreground">Workflow is running...</span>
-                            </div>
-                        )}
-                    </>
-                }
-                leftSidebarOpen={leftSidebarOpen}
+                leftSidebarOpen={false}
```

Comment:
Ako izostaviš ovaj prop, trebalo bi po defaultu bit `false` - boolean propovi su `false` osim ako ih ne definiraš (i to ne triba `={value}` dio nego samo `leftSidebarOpen`)

Reply:
default je true :